lib/religion/religion.ts

```diff
@@ -2565,6 +2634,915 @@ export const religion: ReligionData = {
+          description: 'string',
+          appearance: '',
+          portfolios: [
+            ''
+          ],
+          gender: 'man',
+          shape: 'human',
+          race: 'god',
+          domains: [
+            'tempest'
+          ],
+          channelDivinity: [],
+          alignment: 'N',
+          symbol: '',
+          combat: {
+            description: 'string',
+            weapon: 'string',
+            tactics: 'string'
+          },
+          possessions: [
+            {
+              name: 'string',
+              wordNoun: 'string',
+              powers: 'string'
+            }
+          ],
+          followers: {
+            description: 'string',
+            favouredWeapon: 'spear',
+            holyDays: {
+              earth: ['string']
+            }
+          },
+          personality: {
+            just: 70,
+            vengeful: 85,
+            lustful: 80
+          },
+          associations: {
+            avatars: [
+              {
+                name: 'string',
+                appearance: 'string',
+                description: 'string',
+                frequency: 'string',
+                powers: 'string'
+              }
+            ],
+            animals: [
+              'string'],
+            plants: [
+              'string'
+            ],
+            monsters: [],
+            gems: [],
+            colours: [],
+            miscellaneous: []
+          },
+          beliefs: 'string',
+          heresies: 'string'
+        },
+        { //Hel
+          name: 'Hel',
+          status: 'alive',
+          titles: [
+            ''
+          ],
+          rank: 'greater deity',
+          description: 'string',
+          appearance: '',
+          portfolios: [
+            ''
+          ],
+          gender: 'woman',
+          shape: 'human',
+          race: 'god',
+          domains: [
+            'death',
+            'grave'
+          ],
+          channelDivinity: [],
+          alignment: 'N',
+          symbol: '',
+          combat: {
+            description: 'string',
+            weapon: 'string',
+            tactics: 'string'
+          },
+          possessions: [
+            {
+              name: 'string',
+              wordNoun: 'string',
+              powers: 'string'
+            }
+          ],
+          followers: {
+            description: 'string',
+            favouredWeapon: 'spear',
+            holyDays: {
+              earth: ['string']
+            }
+          },
+          personality: {
+            just: 70,
+            vengeful: 85,
+            lustful: 80
+          },
+          associations: {
+            avatars: [
+              {
+                name: 'string',
+                appearance: 'string',
+                description: 'string',
+                frequency: 'string',
+                powers: 'string'
+              }
+            ],
+            animals: [
+              'string'],
```

Comment:
This looks a bit oddly formatted.

Reply:
It was wrong when I duplicated the template and I didn't want to have to Redo everything when I figured that it would be corrected when it was filled in